api/PublicKeyCredentialCreationOptions.json

```diff
@@ -38,7 +38,7 @@
             "version_added": null
           },
           "webview_android": {
-            "version_added": null
+            "version_added": "67"
```

Comment:
PublicKeyCredential [isn't supported in WebView](https://cs.chromium.org/chromium/src/android_webview/tools/system_webview_shell/test/data/webexposed/not-webview-exposed.txt?l=88). It's creation options wouldn't be either.